package.json

```diff
@@ -1,29 +1,31 @@
 {
-  "name": "dmt",
-  "version": "0.1.0",
-  "description": "Project management tool",
-  "license": "GPL-2.0+",
-  "scripts": {
-    "test": "karma start"
-  },
-  "devDependencies": {
-    "casperjs": "~1.1.0-beta3",
-    "karma": "~0.13.15",
-    "karma-junit-reporter": "~0.3.8",
-    "karma-html2js-preprocessor": "~0.1.0",
-    "karma-phantomjs-launcher": "~0.2.1",
-    "qunitjs": "~1.17.1",
-    "karma-qunit": "~0.1.8",
-    "requirejs": "~2.1.21",
-    "karma-requirejs": "~0.2.2",
-    "karma-sinon": "~1.0.4",
-    "phantomjs": "~1.9.18",
-    "sinon": "~1.17.2",
-    "sinon-qunit": "~2.0.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/ccnmtl/dmt.git"
-  },
-  "readmeFilename": "README.markdown"
+    "name": "dmt",
```

Comment:
Hmm. Yeah, my emacs is configured for our usual 4-spaces. I don't have a good sense yet of whether we'll be manually tweaking package.json files more often than using `npm --save`. I don't think we'll be doing either terribly often so it's probably not a big deal either way.